src/renderer/src/views/components/Files/__tests__/tabIndex.test.ts

```
@@ -0,0 +1,209 @@
+  type Handler = (...args: any[]) => void
+  const handlers = new Map<string, Set<Handler>>()
+  return {
+    on: vi.fn((event: string, fn: Handler) => {
+      if (!handlers.has(event)) handlers.set(event, new Set())
+      handlers.get(event)!.add(fn)
+    }),
+    off: vi.fn((event: string, fn: Handler) => {
+      handlers.get(event)?.delete(fn)
+    }),
+    emit: vi.fn((event: string, ...args: any[]) => {
+      handlers.get(event)?.forEach((fn) => fn(...args))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)